### DISCUSSION TITLE
Please help me with the system, I have lost the connection with the docker or the system stopped working after an upgrade

### DISCUSSION THREAD

**Discussion by miguel2962** [microrealestate/microrealestate#197] [General]
I updated the system on my Ubuntu server in the AWS domain and did not delete the Dockers so as not to lose information, I only deleted and cloned and git again and ran mre build and it gives me errors and if I use mre start I start again and select the first option but it does not let me create or login with my credentials from my previous database.

![3](https://github.com/user-attachments/assets/69afdaac-303e-4e44-afb5-85e409a15afa)
![Capture](https://github.com/user-attachments/assets/24d56250-0caf-49b6-8102-b9ce2e8b02e6)
![Capture1](https://github.com/user-attachments/assets/4463e108-4856-4d1f-b067-1a0c74b033ac)
![Capture2](https://github.com/user-attachments/assets/64dd8791-ea11-4e17-bf7d-b64b96934a1d)

**Comment by camelaissani** [comment]
@miguel2962 can you ping me on LinkedIn?

**Reply by miguel2962** [reply]
I see that to use LinkedIn you have to pay to send messages. If you want to use another social network, Telegram.

**Reply by camelaissani** [reply]
https://x.com/camelaissani

**Reply by miguel2962** [reply]
I see that to use X you have to pay to send messages if you want to use another social

**Reply by camelaissani** [reply]
Sorry I haven't others social networks.

Did you backup your data before using  the new version?

**Reply by miguel2962** [reply]
I applied the mre dumpdb command and it told me that it could not find dev but I am in production mode for that reason I did not delete the dockers that the system creates

**Comment by miguel2962** [comment]
I applied the mre dumpdb command and it told me that it could not find dev but I am in production mode for that reason I did not delete the dockers that the system creates


Best Regards,



Miguel Angel Milton Florian

systems engineer

 Oranjestad, Aruba

Mobile phone: +(297) 594-9417


________________________________
De: camelaissani ***@***.***>
Enviado: Monday, September 16, 2024 3:02:08 PM
Para: microrealestate/microrealestate ***@***.***>
Cc: miguel2962 ***@***.***>; Mention ***@***.***>
Asunto: Re: [microrealestate/microrealestate] Please help me with the system, I have lost the connection with the docker or the system stopped working after an upgrade (Discussion #197)


Sorry I haven't others social networks.

Did you backup your data before using the new version?
